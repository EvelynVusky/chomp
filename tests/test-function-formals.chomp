void foo(int a, string s, bit b, nibble n, bool bo, char c) {
    println(a);
    println(s);
    println(b);
    println(n);
    println(bo);
    println(c);
} 

int main() {
    int a = 66;
    string s = "hi!";
    bit b = {{1}};
    nibble n = {{0101}};
    bool bo = true;
    char c = '!';
    foo(a, s, b, n, bo, c);
    return 0;
}