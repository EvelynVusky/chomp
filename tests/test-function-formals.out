66
hi!
1
0101
1
!
